interfaces/vrep/DQ_SerialVrepRobot.m

```diff
@@ -0,0 +1,242 @@
+% CLASS DQ_SerialVrepRobot - Concrete class to interface with serial robots
+% in VREP.
+%
+% Usage:
+%       1) Drag-and-drop a serial robot to a VREP scene. For instance, a
+%       "my_robot" robot.
+%       2) Run
+%           >> vi = DQ_VrepInterface();
+%           >> vi.connect('127.0.0.1',19997);
+%           >> vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot", vi);
+%           >> vi.start_simulation();
+%           >> vrep_robot.get_configuration_space_positions();
```

Comment:
Let's change it to `get_configuration()` and deprecate `get_configuration_space_positions()`.

Reply:
Updated comment in [c934c9c](https://github.com/dqrobotics/matlab/pull/108/commits/c934c9cca0aed4d1327b7c9c18b07047b580702a). However, there is no need to deprecate `get_configuration_space_positions()` as this is the pull requesting proposing the first MATLAB version of `DQ_SerialVrepRobot`. We willl have to deprecate the methods in the [C++ version](https://github.com/dqrobotics/cpp-interface-vrep/blob/master/include/dqrobotics/interfaces/vrep/DQ_SerialVrepRobot.h), though.

Reply:
@ffasilva nothing only the `master` branch needs to be deprecated. Only what ended up in a release version.